modules/rosa-delete-objects.adoc

```diff
@@ -75,6 +75,9 @@ $ rosa delete cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 
 |--watch
 |Watches the cluster uninstallation logs.
+
+|--best-effort
+|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. This option should be used with care and may require manual cleanup of certain resources.
```

Comment:
```suggestion
|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. This option should be used with care and might require manual cleanup of certain resources.
```

Reply:
| cleanup of certain resources

How do I determine if manual cleanup is needed, and what needs cleaning up?

Reply:
> | cleanup of certain resources
> 
> How do I determine if manual cleanup is needed, and what needs cleaning up?

Good question, and one I have been trying to get an answer to. Let me check again.